addons/knowledge/models/knowledge_article.py

```diff
@@ -726,22 +726,25 @@ def action_unarchive(self):
     # SEQUENCE / ORDERING
     # ------------------------------------------------------------
 
-    def move_to(self, parent_id=False, before_article_id=False, is_private=False):
+    def move_to(self, parent_id=False, before_article_id=False, is_private=False, resequence_shared_root=False):
```

Comment:
Quid if we replace the `is_private` with the name of the new category name ('workspace', 'shared', 'private') ? We would then have 3 arguments instead of 4. We would also do less gymnastic in the javascript :smiley:

Reply:
@dbeguin

Reply:
this could make the code a bit more readable and structured, indeed. 
Let's do that. :+1: